src/module/compcon.ts

```diff
@@ -0,0 +1,33 @@
+import { PackedPilotData } from 'machine-mind';
+import Auth from "@aws-amplify/auth";
+import Storage from "@aws-amplify/storage";
+
+// we don't cache anything other than id/name; we're going to fetch all other data on user input
+// but we don't want the pilot actor window to wait for network calls
+// this does mean that the GM needs to refresh foundry to clear this cache if they add pilots
+// (they could also re-login, we initiate a cache refresh there)
```

Comment:
Maybe we can add a refresh button next to the vault pilot selector on the pilot sheet. That would be a nice-to-have, can be done later.